onnxruntime/core/framework/graph_partitioner.cc

```diff
@@ -80,12 +80,10 @@ Status GraphPartitioner::Partition(Graph& graph, bool export_dll, FuncManager& f
   // It is only visible for current session.
   std::shared_ptr<KernelRegistry> fused_kernel_registry = std::make_shared<KernelRegistry>();
   // Partitioning <graph> based on provider preference and their capabilities.
-  auto kernel_registries = kernel_registry_mgr_.GetAllKernelRegistries();
-
   std::vector<std::vector<std::unique_ptr<ComputeCapability>>> capabilities_of_all_providers;
   GraphViewer graph_viewer(graph);
   for (auto& provider : providers_) {
-    capabilities_of_all_providers.push_back(provider->GetCapability(graph_viewer, kernel_registries));
+    capabilities_of_all_providers.push_back(provider->GetCapability(graph_viewer, kernel_registry_mgr_.GetKernelRegistriesByProviderType(provider->Type())));
   }
```

Comment:
nit: line too long. 120 is the max. [https://marketplace.visualstudio.com/items?itemName=PaulHarrington.EditorGuidelines](https://marketplace.visualstudio.com/items?itemName=PaulHarrington.EditorGuidelines) is very helpful as you can set a guideline at 120 and see if something is too long.

It's not enforced by clang-format so that developers can choose where to put line breaks for better readability (vs. clang-format always trying to fit as much as possible on a line).